app/views/layouts/application.html.haml

```diff
@@ -43,7 +43,7 @@
               %ul
                 %li
                   %a{href: "/users/edit"} Settings
-            - unless @projects.empty?
+            - unless @projects.empty? && current_user.followed_projects.empty?
               %section
```

Comment:
Because of this line, if user had no projects his side bar won't be visible even if he had some followed_projects.